application-workloads/sharepoint/sharepoint-adfs/azuredeploy.json

```diff
@@ -373,7 +378,34 @@
       "spADDirSyncUserName": "spdirsync",
       "localAdminUserName": "[concat('local-', uniqueString(subscription().subscriptionId))]",
       "enableAnalysis": false,
-      "isSharePointvNext": "[if(startsWith(parameters('sharePointVersion'), 'subscription'), true(), false())]"
+      "isSharePointSubscription": "[if(startsWith(parameters('sharePointVersion'), 'subscription'), true(), false())]",
+      "sharePointSubscriptionBits" : [
+        {
+          "Label": "RTM",
+          "Packages": [
+            {
+              "DownloadUrl": "https://go.microsoft.com/fwlink/?linkid=2171943",
```

Comment:
can we use a non-fwlink for this as well?  it ensures the template cannot change without users knowing...

Reply:
I understand, I just updated the link as requested